corehq/apps/hqwebapp/static/cloudcare/less/formplayer-webapp/form.less

```diff
@@ -39,6 +39,25 @@
   .panel-body {
     padding-left: @form-text-indent + 5px;
     padding-right: @form-text-indent + 5px;
+    // Bootstrap introduces -10px left/right margin for row classes. This causes element to overflow parent.
```

Comment:
this would affect non-small screens, right? Might be overkill, but we may need to flag prior to deploy to other divisions in that case.

Reply:
Always a good thing to bring up. But I'll go on record saying this one is minor enough not to formally communicate to other divisions.

Related, all of our responsive design changes affect other divisions. USH doesn't own web apps on small screens. But you're definitely right that a non-small-screens change is more likely to be noticed.

Reply:
Thanks for noting - I also think this is minor enough. But for future reference, what's the formal communication process?  I remember there was a change communication doc or something, but can't find it

Reply:
cool makes sense -- just wanted to err on the safe side by flagging  👍🏻

Reply:
@Jtang-1 check out this: https://confluence.dimagi.com/display/saas/Change+Management+Comms